addons/addon-base-raas/packages/base-raas-services/lib/data-egress/__test__/data-egress-service.test.js

```diff
@@ -254,6 +254,9 @@ describe('DataEgressService', () => {
           if (settingName === 'egressStoreKmsKeyAliasArn') {
             return 'test-egressStoreKmsKeyAliasArn';
           }
+          if (settingName === 'awsPartition') {
+            return 'aws';
```

Comment:
how about adding test cases for partition `aws-cn`?

Reply:
This settings is a config, I think it is fine to check the result string value is from config.